src/watcher.js

```diff
@@ -45,9 +45,15 @@ export default function onSourceChange(
     debounceTime = 1000,
   }: OnSourceChangeParams
 ): Watchpack {
+  // When running on windows, transform the ignored paths and globs
+  // as Watchpack does translate the changed files path internally.
```

Comment:
Could you link to the relevant code for future reference?

E.g. https://github.com/webpack/watchpack/blob/v2.1.1/lib/DirectoryWatcher.js#L99-L103